src/Params/LFOParams.h

```diff
@@ -69,6 +69,9 @@ class LFOParams:public Presets
         float         fadein; /**<fadein, relative to delay*/
         float         fadeout; /**<fadeout on key release (10.0=off)*/
         unsigned char Pcontinous; /**<1 if LFO is continous*/
+        float         speedratio; /**<ratio between system tempo and LFO freq (0=off)*/
```

Comment:
Still unused. Can you remove it, please?

Reply:
ok